lms/envs/bok_choy.py

```diff
@@ -232,16 +232,7 @@
         'course_author': 'http://edx.readthedocs.io/projects/edx-partner-course-staff',
     }
 
-# TODO: TNL-6546: Remove this waffle and flag code.
-from django.db.utils import ProgrammingError
-from waffle.models import Flag
-try:
-    flag, created = Flag.objects.get_or_create(name='unified_course_view')
-    WAFFLE_OVERRIDE = True
-except ProgrammingError:
-    # during initial reset_db, the table for the flag doesn't yet exist.
-    pass
-
+WAFFLE_OVERRIDE = True
```

Comment:
@efischer19 Hey Eric.  My block was a hack that was not a great idea in the first place.  I'd say anything is fine as long as:
1. You leave a TODO anywhere the flag is explicitly mentioned, and
2. The bokchoy tests continue to run and pass.

Thanks.